sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -376,7 +376,7 @@ def replace_item(
         post_trigger_include=None,  # type: Optional[str]
         **kwargs  # type: Any
     ):
-        # type: (...) -> Dict[str, str]
+        # type: (...) -> Dict[str, Any]
```

Comment:
Do we have tests to validate after the change, no following errors?
`With dict[str, str] as a return type you will get type errors when using nesting in json.`

Reply:
We don't have any particular type checks for tests but If dict[str, str] wasn't breaking anything, this won't either (str falls under Any as you would expect). Creating a test for this would be weird as well, since it might be a first thought to parse through the dict and check for the values having different types, but a user could also choose to have it all be strings mapped to only strings.

I also ran the existing tests and samples with this and had same results as before, so we should be good!